addon/components/environment-banner.js

```diff
@@ -11,8 +12,13 @@ const InfoCircleIcon = macroCondition(
       .InfoCircleIcon
   : 'info-circle';
 
+const or = helper(function ([a, b]) {
```

Comment:
I went with a local helper instead of installing ember-truth-helpers, to not have to deal with peerDeps / version conflicts, etc.

I think we ideally deprecated the `@message` version, so after removal we would no longer need this helper either.